ci.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+set -e 
+set -x 
+
+# Get storage drive details
+docker info
+
+# Create .dockercache directory
+mkdir -p ./.dockercache/
+
+# Import devcontainer from cache to speed up build
+if [ -f ".dockercache/devcontainer.tar" ]; 
```

Comment:
Could we have this script as an inline task instead of having a separate file? having separate file looks cleaner but given it's a public repo I would like to make sure we don't accidentally overlook file changes related to the build pipeline in PRS

Also, I was thinking maybe we could use `condition: eq(variables.DOCKER_CACHE_HIT, 'true')` instead of checking the cached file?

```
- task: Cache@2
    inputs:
      key: docker-image | .devcontainer/**
      path: ".dockercache"
      restoreKeys: docker-image
      cacheHitVar: DOCKER_CACHE_HIT
    displayName: Cache docker layers
- script: 
  docker info
  docker load -i .dockercache/devcontainer.tar
  ....
  condition: eq(variables.DOCKER_CACHE_HIT, 'true')


```

Reply:
have moved the build script elements back into the yaml as suggested :+1: